Tests/ContributeWordPressTests/XCTestCase+Extension.swift

```diff
@@ -0,0 +1,148 @@
+import XCTest
+import SyndiKit
+@testable import ContributeWordPress
+
+extension XCTestCase {
```

Comment:
I would put these as extensions on the types they return such as `WordPressPost`. I assume they used in more then one test